src/aat/resources/features/F-1005 - Start Case Event TTL/S-1005.1.td.json

```diff
@@ -0,0 +1,72 @@
+{
+  "_guid_": "S-1005.1",
+  "_extends_": "Befta_Jurisdiction2_Token_For_Update_Case",
+  "title": "must successfully update a cases TTL.SystemTTL value",
```

Comment:
Title in scenario TD files should match feature file's corresponding scenario description.

Reply:
Will be updated by 69f4c8d387474245b16c099ef1a8b7eddf556fac in #1974.